tests/Builder/input/index.rst

```diff
@@ -19,6 +19,10 @@ Link to :ref:`the same doc reference <same_doc_reference>`
 
 .. _same_doc_reference:
 
+Link to :ref:`the same doc reference with ticks <same_doc_reference_ticks>`
+
+.. _`same_doc_reference_ticks`:
```

Comment:
This HTML needs to be asserted somewhere.

Reply:
Yep, of course :)